dashboard/src/App.js

```diff
@@ -92,23 +91,11 @@ const App = () => {
                       path={APP_ROUTES.TABLE_OF_CONTENT}
                       element={<TableOfContent />}
                     />
```

Comment:
This isn't really a "proper" comment on this PR, and it probably ought to have been a separate Jira (which I continually forget to write) ... but it's a bit annoying to be looking at the Results page, without login, click on a public dataset, and be asked to log in before it'll show me the public dataset's TOC. Should anything other than the profile dialog and the Overview page really be "protected"??